Orange/projection/freeviz.py

```diff
@@ -0,0 +1,463 @@
+
+import numpy as np
+import scipy.spatial
+
+from Orange.data import ContinuousVariable, Domain
+from Orange.data.util import SharedComputeValue
+from Orange.projection import Projector, Projection
+
+
```

Comment:
Define `__all__` here and pull `FreeViz` up into Orange.projection.
Tests and documentation should then import from there.